### PR TITLE
feat: MyCart

### PR DESCRIPTION
**Pull request by prakash855**

### CODE REVIEW

**Review by palash23jain — APPROVED** [review]
good naming of classes. looks good to go

**Comment by RakshaPawar108** [comment]
Consider adding a description to your PR so that reviewer understands what the PR is about and add screenshot or gif if possible

File: component/MyCart/MyCart.html [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+    <link rel="stylesheet" href="../../../style.css">
+    <link rel="stylesheet" href="./MyCart.css">
+    <title>eBuy</title>
+</head>
+
+</style>
+
+<body>
+    <div class="home_container">
+        <nav class="navigation flex shadow">
```
Comment (RakshaPawar108) [inline component/MyCart/MyCart.html:24]: Good use of semantic HTML!

**Review by RakshaPawar108 — APPROVED** [review]
Looks good to merge!

**Pull request merged**